scss/modules/form.scss

```diff
@@ -360,7 +360,7 @@ textarea.ckeditor-preview {
             height: $radioSize;
             padding: 2px;
             border-radius: $radioSize;
-            border: 1px solid $borderColor;
+            border: 1px solid $black;
```

Comment:
Why are we changing the value here? Why don't we use the more explicit `$borderColor` variable anymore?